### DISCUSSION TITLE
Last chat update break the placeholder

### DISCUSSION THREAD

**Discussion by Darkseno** [Zrips/CMI#7813] [Enhancement]
### Description of Issue

Hello, I have been using the same chat configuration for more than 4 months now and I have never had a problem before and since your last update everything is broken.

Here is the configuration I was using for all the groups:

     '14': '&7* &r%img_Founder% %luckperms_meta_username-color%%cmi_user_nickname% &8%robottags_player%&8 &8"
      &f{message}'

The little star is the FactionUUID plugin, it has always allowed to display the name of the factions of the players with the rank and especially the color of their relationship, since it does not work at all, it just displays a star in game.

        tagReplaceString="*" (in the faction plugin)

I said to myself no problem I'm going to use REAL placeholder, so I used this: 

        %rel_factionsuuid_relation_color%%factionsuuid_faction_name_custom%

But again it doesn't work, it only displays the faction name in white and doesn't take the color into account.

I would like to point out that I use these placeholders on other scoreboard or menu plugins and all of them work fine, I also tried a /papi parse and all of them are working so I would like to understand what you broke?

If possible I would like to use my star again because it allows to display the factions, according to the world etc....

### Version Information

```shell
[08:06:06 INFO]: --------------------------------------------------
[08:06:06 INFO]: CMI: 9.5.0.6 BungeeCord  MySQL
[08:06:06 INFO]: CMILib: 1.2.5.2 
[08:06:06 INFO]: Server: Purpur 1.19.4-R0.1-SNAPSHOT
[08:06:06 INFO]: CMI economy: Enabled Vault: 1.7.3-CMI CMI Chat: Enabled 
[08:06:06 INFO]: Modules -> 29 enabled 29 disabled: spawnerProximity, tablist, moneyCheque, skin, noTarget, elytraBoost, cmiPlaytime, eventCommands, elytraLaunch, playerCombat, votifier, shulkerBackpack, headDrop, attachedCommands, spawnerCharge, timedCommands, launchPad, portals, anvilRenameColor, spawners, paintingEditor, dynamicSigns, durabilityLoss, kits, flightCharge, schedule, worldLimits, ranks, silentChest
[08:06:06 INFO]: --------------------------------------------------
```

### Errors

```shell
No errors
```

### Relevant Plugins

 Vault 1.7.3-C.M.I 
 FactionsUUID 1.6.9.5-U0.6.29-b322
 PlaceholderAPI 2.11.3
 Purpur - 1953 (1.19.4)

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by CloudeLecaw** [comment]
may am i wrong but if i renmeber correct the 
%rel_factionsuuid placeholders and the * one where never officialy supprted...
if it was working... then it might was more luck then a working.. so it doesnt matter for now if it WAS working or not..
well corecct me if i was wrong about this @Zrips 

I dont know how much server using this but it might worth to add a request for official support for these placeholders in cmi githubs discussion
https://github.com/Zrips/CMI/discussions

**Comment by Darkseno** [comment]
> may am i wrong but if i renmeber correct the %rel_factionsuuid placeholders and the * one where never officialy supprted... if it was working... then it might was more luck then a working.. so it doesnt matter for now if it WAS working or not.. well corecct me if i was wrong about this @Zrips
> 
> I dont know how much server using this but it might worth to add a request for official support for these placeholders in cmi githubs discussion https://github.com/Zrips/CMI/discussions

There is no such thing as luck. It worked for 1 year, I don't see why today it's broken, I just want to understand why today.

**Comment by CloudeLecaw** [comment]
belive it or not ;)

Other used cmi some years with Mohist or Velocity and never aspected it might be not supported.

feel still free to suggest offical support for it here.
https://github.com/Zrips/CMI/discussions

**Comment by Darkseno** [comment]
UP ?

**Comment by CloudeLecaw** [comment]
it does not need to be up *lol*

Up-threads should be removed ^^

Pls move this to discussion because it sounds more like a request for official support of %rel placeholders

**Comment by Zrips** [comment]
This is not a bug on itself, at the moment we are not handling relational placeholders. This might come down the line.

**Reply by Darkseno** [reply]
> This is not a bug on itself, at the moment we are not handling relational placeholders. This might come down the line.

How do you explain that a single star in my chat config was working fine?

**Comment by SbirTeo** [comment]
Please. News?
